_data/students.yml

```diff
@@ -11,6 +11,14 @@
 #   lat: Student's Latitude
 #   lng: Student's Longitude
 
+
+ - name: Shivam Rai
+   country: India
+   github: ShivamRai2003
+   image: (rai)shivam.jpg
```

Comment:
Please upload an image by this name.

Reply:
please upload it so i my task should be reviwed

Reply:
please update my profile to website

Reply:
isuruAb where is the commits are available

Reply:
i have changed all